code-challenges.test.js

```diff
@@ -52,5 +144,67 @@ const hand2 = [5, 5, 3, 3, 4]
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe("fullHouseChecker", () => {
+    it("Takes in an array of 5 numbers and checks for one pair and three of a kind", () => {
+      expect(fullHouseChecker(hand1)).toEqual(true)
+      expect(fullHouseChecker(hand2)).toEqual(false)
+      expect(fullHouseChecker(hand3)).toEqual(false)
+    })
+  })
+
+//   FAIL  ./code-challenges.test.js
+//   fullHouseChecker
+//     ✕ Takes in an array of 5 numbers and checks for one pair and three of a kind (1 ms)
+
+//   ● fullHouseChecker › Takes in an array of 5 numbers and checks for one pair and three of a kind
+
+//     ReferenceError: fullHouseChecker is not defined
 
 // b) Create the function that makes the test pass.
+
+// pseudo code:
+// create function that takes in array of 5 numbers
+// create an empty object
+// itterates through the array
+// check if value in object doesn't exist
+// if yes define as key in object, if not move on to next step
+// add + 1 to the value of the key which is named after the number in the array
+// when itteration function done, check if there are 3 of a kind for one key and a pair for the other
+// return if true or false
+
+const fullHouseChecker = (array) => {
+    var countHolder = {}
+    for (let i = 0; i < array.length; i++){
```

Comment:
Good for loop and conditional statement.  It still works but try using 'let' instead of 'var' from now on.  Var can create several bugs in a big project, FYI.  👍🏻

Reply:
thanks, i'll keep this in mind!